src/settings/LanguageDialog.tsx

```diff
@@ -70,6 +70,10 @@ export const LanguageDialog = ({ isOpen, onClose }: LanguageDialogProps) => {
                 <Icon as={RiExternalLinkLine} />
               </Link>
             </VStack>
+            <Text fontSize="xs" alignSelf="flex-end" mt={1}>
+              * These languages are an early preview of in-progress
```

Comment:
Maybe this should now have the "intentionally not translated" comment.

Reply:
Makes sense, though I'll revisit and/or remove this before launch so not going to bother now.